src/Images.jsx

```diff
@@ -232,6 +232,22 @@ class Images extends React.Component {
 
         const imageRows = filtered.map(id => this.renderRow(this.props.images[id]));
 
+        const sortRows = (rows, direction, idx) => {
+            // Image / Owner / Created / ID / Disk space / Used by
+            const isNumeric = idx === 2 || idx === 4 || idx === 5;
+            const sortedRows = rows.sort((a, b) => {
+                const aitem = a.columns[idx].sortKey ?? a.columns[idx].title;
+                const bitem = b.columns[idx].sortKey ?? b.columns[idx].title;
```

Comment:
These 2 added lines are not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2239-eb55a47a-20250828-172217-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)